src/app/faq/page.tsx

```diff
@@ -40,7 +40,7 @@ export default function Faq() {
         />
         <QAComponent
           question='活動内容を教えてください！'
-          answer='競技プログラミングやWeb制作・Webパフォーマンスチューニングなどを行っています！詳細は上のAboutページをご覧ください！'
+          answer='競プロやWeb研を中心に、2025年度からはCTF・広義AI・モバイルアプリなどの新たな活動班も加わり、幅広い分野に挑戦できるサークルになりました！詳細は上のAboutページをご覧ください！'
```

Comment:
```suggestion
          answer='競技プログラミングやWeb研究会を中心に、2025年度からはCTF・広義AI・モバイルアプリなどの新たな活動班も加わり、幅広い分野に挑戦できるサークルになりました！詳細は上のAboutページをご覧ください！'
```
初見の人向けのサイトなので省略せずに書いた方がいいかも

Reply:
了解です！